sources/tools/Stride.StorageTool/Stride.StorageTool.csproj

```diff
@@ -1,23 +1,26 @@
-<Project>
-  <Import Project="..\..\targets\Stride.props" />
```

Comment:
It should be possible to keep importing our .props file even when working with Avalonia.